docs/source/en/pipeline_tutorial.mdx

```diff
@@ -33,101 +33,295 @@ While each task has an associated [`pipeline`], it is simpler to use the general
 ```py
 >>> from transformers import pipeline
 
->>> generator = pipeline(task="text-generation")
+>>> generator = pipeline(task="automatic-speech-generation")
 ```
 
 2. Pass your input text to the [`pipeline`]:
 
 ```py
 >>> generator(
-...     "Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone"
-... )  # doctest: +SKIP
-[{'generated_text': 'Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone, Seven for the Iron-priests at the door to the east, and thirteen for the Lord Kings at the end of the mountain'}]
+...     "https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/mlk.flac"
```

Comment:
Is there any way to have a widget/link to a widget where the user could listen to the sample?

Reply:
Well if you use this link it should play directly in your browser. 
Does so for FF at least. (It's  a hub configuration away if it doesn't)